build.sbt

```diff
@@ -102,6 +102,16 @@ lazy val aws = (project in file("spade-aws"))
 lazy val examples = (project in file("spade-examples"))
   .settings(commonSettings: _*)
   .settings(noPublishSettings: _*)
+  .settings(assembly / test := {})
```

Comment:
why need to assemble a lib?

Reply:
It is for the spade-examples, ExampleWorkflow.  When testing sprinkler, I needed to have a jar for that to run.  

ref: https://github.com/salesforce/spade/pull/18